magprime/templates/static_views/star_rod_wand.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+<head>
+  <title>Star Rod Wand</title>
+  <style type="text/css">
+    html, html * { font-family: Arial, Helvetica, sans-serif; }
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>Star Rod Wand</h1>
+
+<p>
+  What kind of a SUPER STAR would be caught off guard without their trusty
+  star rod wand to defend themselves?
+</p>
+<p>
+  This custom-made plastic molded stick of fabulousness is manufactured
+  exclusively for MAGFest 2018, and is available exclusively for purchase by
```

Comment:
All of your suggestions, except for this one, were made in this pull request: https://github.com/magfest/magprime/pull/222.

I didn't change this text because that's explicitly what Nick wanted. I was also a little surprised by the wording. Maybe @nickthenewbie1 can review this when he gets back from vacation.